### ISSUE TITLE
Sprite Distorsion

### ISSUE THREAD

**Issue by gr1ndy** [notgoodusename/OsirisAndExtra#435]
https://user-images.githubusercontent.com/71841362/208241160-06e3c592-2cce-4b60-a1c1-9f0da1b1fa43.mp4

So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.

What could be causing this?

**Comment by gr1ndy** [comment]
https://user-images.githubusercontent.com/71841362/208244618-a9ccd317-cb7c-4d84-acae-4e3517d51a6d.mp4

same thing is happening with bullet tracers aswell

**Comment by Xsintashi** [comment]
bullet traces are distorting cuz they are animated 

https://user-images.githubusercontent.com/61426269/208245155-75e0475f-54c4-4702-b88f-982347332a77.mp4

**Comment by gr1ndy** [comment]
> bullet traces are distorting cuz they are animated
> 
>  csgo.2022-12-17.14-47-41-028.mp4

still, they have no business distorting like that.

**Comment by rrhyast** [comment]
> Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> 
> So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> 
> What could be causing this?

can you pull request sound esp please?

**Comment by gr1ndy** [comment]
> > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > What could be causing this?
> 
> can you pull request sound esp please? + exclude from spotted only esp option

it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw

**Comment by rrhyast** [comment]
> > > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > > What could be causing this?
> > 
> > 
> > can you pull request sound esp please? + exclude from spotted only esp option
> 
> it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw

also please make it visible behind the walls.

**Comment by gr1ndy** [comment]
> > > > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > > > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > > > What could be causing this?
> > > 
> > > 
> > > can you pull request sound esp please? + exclude from spotted only esp option
> > 
> > 
> > it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw
> 
> also please make it visible behind the walls.

iirc, thats based on radius..

**Comment by rrhyast** [comment]
> > > > > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > > > > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > > > > What could be causing this?
> > > > 
> > > > 
> > > > can you pull request sound esp please? + exclude from spotted only esp option
> > > 
> > > 
> > > it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw
> > 
> > 
> > also please make it visible behind the walls.
> 
> iirc, thats based on radius..

that's correct, When I used to play css v34, there was a cheat called ultra-hook with a lots of customization and it contains sound esp that was visible behind the walls. when it's possible on 19 years old game based on same engine, why not using it?

**Comment by gr1ndy** [comment]
> > > > > > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > > > > > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > > > > > What could be causing this?
> > > > > 
> > > > > 
> > > > > can you pull request sound esp please? + exclude from spotted only esp option
> > > > 
> > > > 
> > > > it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw
> > > 
> > > 
> > > also please make it visible behind the walls.
> > 
> > 
> > iirc, thats based on radius..
> 
> that's correct, When I used to play css v34, there was a cheat called ultra-hook with a lots of customization and it contains sound esp that was visible behind the walls. when it's possible on 19 years old game based on same engine, why not using it?

i've added a small list of sprites that u can use for footstep esp. you'll see when ill make the pull tomorrow.

**Comment by gr1ndy** [comment]
> > > > > > Counter-Strike_.Global.Offensive.-.Direct3D.9.2022-12-17.14-08-44.mp4
> > > > > > So i added foot step ESP for more visual customisation and I found that if the sprite radius is over 4-5 it'll start doing weird things before turning into a ring.
> > > > > > What could be causing this?
> > > > > 
> > > > > 
> > > > > can you pull request sound esp please? + exclude from spotted only esp option
> > > > 
> > > > 
> > > > it isnt spotted only, you can place it wherever u want. ill do the pull tmrrw
> > > 
> > > 
> > > also please make it visible behind the walls.
> > 
> > 
> > iirc, thats based on radius..
> 
> that's correct, When I used to play css v34, there was a cheat called ultra-hook with a lots of customization and it contains sound esp that was visible behind the walls. when it's possible on 19 years old game based on same engine, why not using it?

https://github.com/notgoodusename/OsirisAndExtra/pull/439
